test/unittests/audio/test_speech.py

```diff
@@ -82,6 +93,118 @@ def test_stop(self, check_for_signal_mock, tts_factory_mock, config_mock):
         self.assertNotEqual(speech._last_stop_signal, 0)
         speech.shutdown()
 
+    def test_fallback_tts_creation(self, tts_factory_mock, config_mock):
+        """Ensure the init and shutdown behaves as expected."""
+        setup_mocks(config_mock, tts_factory_mock, fallback="B")
+        bus = mock.Mock()
+        speech = PlaybackService(bus=bus)
+        speech.fallback_tts = None
+        speech._get_tts_fallback()
+        self.assertIsNotNone(speech.fallback_tts)
```

Comment:
should this be `tts_mock` and can we check equality or `isinstance` here?

Reply:
in step above i explicitly assign it to None, it would have been a tts_mock already from the init step, so i thought this made it clearer, the check really is "is this not None anymore?"